src/components/Globalstats.js

```diff
@@ -1,32 +1,40 @@
-/* eslint-disable */
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowRight } from 'react-icons/fa';
+import { Spinner } from 'react-bootstrap';
 import { fetchCovidData } from '../redux/statisticsSlice';
-import '../App.css'
-import { showDetails } from '../redux/statisticsSlice';
+import CountryStats from './CountryStats';
+import '../App.css';
 
 const Globalstats = () => {
-  const { dataArray/*, isLoading */ } = useSelector((state) => state.Covid);
-    const dispatch = useDispatch();
-  
-  const handleClick = (country) => {
-    dispatch(showDetails(country))
-  }  
-    if (dataArray.length === 0) {
-        setTimeout(() => {
-          dispatch(fetchCovidData());
-        }, '1000');
-      }
-    
+//  const [query, setQuery] = useState('');
+  // const Search = (event) => {
+  //   setQuery(event.target.value);
+  // };
```

Comment:
- [ ] Please remove any commented code to make you code clean base and dry since if you don't need it just remove it and please remove all the commented code in your project 🔥